app/models/vm_or_template.rb

```diff
@@ -1193,7 +1193,7 @@ def refresh_on_reconfig
     unless ext_management_system.authentication_status_ok?
       raise _("%{table} failed last authentication check") % {:table => ui_lookup(:table => "ext_management_systems")}
     end
-    EmsRefresh.reconfig_refresh(self)
+    EmsRefresh.refresh(self)
```

Comment:
If you are replacing this (which I'm not sure you can, but *if*), then this entire refresh_on_config method can be deleted, and the event handling can also be replaced / deleted.